spec/checkPullRequestJobSpec.js

```diff
@@ -0,0 +1,410 @@
+        owner: payloadData.payload.repository.owner.login,
+        labels: ['critical']
+      });
+    });
+  });
+
+  describe('When pull request creates multiple job files', () => {
+    beforeEach(async () => {
+      github.pulls = {
+        listFiles: jasmine.createSpy('listFiles').and.resolveTo({
+          data: [
+            {
+              filename: 'core/templates/App.ts',
+            },
+            newJobFileObj,
+            secondNewJobFileObj
+          ],
+        }),
+      };
+
+      await robot.receive(payloadData);
+    });
+
+    it('should check for jobs', () => {
+      expect(checkPullRequestJobModule.checkForNewJob).toHaveBeenCalled();
+    });
+
+    it('should get modified files', () => {
+      expect(github.pulls.listFiles).toHaveBeenCalled();
+    });
+
+    it('should ping server jobs admin', () => {
+      expect(github.issues.createComment).toHaveBeenCalled();
+      const author = payloadData.payload.pull_request.user.login;
+      const formText = 'server jobs form'.link(
+        'https://goo.gl/forms/XIj00RJ2h5L55XzU2');
+      const newLineFeed = '<br>';
+      const wikiLinkText = (
+        'This PR can be merged only after the test is successful'.link(
+          'https://github.com/oppia/oppia/wiki/Running-jobs-in-production' +
+          '#submitting-a-pr-with-a-new-job'));
+
+      expect(github.issues.createComment).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        body: 'Hi @' + SERVER_JOBS_ADMIN + ', PTAL at this PR, ' +
+        'it adds new one off jobs. The jobs are exp_jobs_one_off, exp_jobs_oppiabot_off.' +
+        newLineFeed + 'Also @' + author + ', please add the new job ' +
+        'files to the job registry and please make sure to fill in the ' +
+        formText + ' for the new jobs  to be tested on the backup server. ' +
+        wikiLinkText + '.' + newLineFeed + 'Thanks!',
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+      });
+    });
+
+    it('should assign server jobs admin', () => {
+      expect(github.issues.addAssignees).toHaveBeenCalled();
+      expect(github.issues.addAssignees).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+        assignees: [SERVER_JOBS_ADMIN]
+      });
+    });
+
+    it('should add critical label', () => {
+      expect(github.issues.addLabels).toHaveBeenCalled();
+      expect(github.issues.addLabels).toHaveBeenCalledWith({
+        issue_number: payloadData.payload.pull_request.number,
+        repo: payloadData.payload.repository.name,
+        owner: payloadData.payload.repository.owner.login,
+        labels: ['critical']
+      });
+    });
+  });
+
+  describe('When pull request creates a new job file and updates registry', () => {
```

Comment:
`When pull request creates a new job file and updates registry` -> `When  a new job file is created and registry is updated in a pull request`